src/Components/README.md

```diff
@@ -29,29 +29,41 @@ The following contains a description of each sub-directory in the `Components` d
 
 ## Development Setup
 
-### Build
+**Note**: To build other specific projects from source, follow the instructions [on building the project](../../docs/BuildFromSource.md#step-3-build-the-repo).
 
-To build this specific project from source, follow the instructions [on building the project](../../docs/BuildFromSource.md#step-3-build-the-repo).
+### Building ASP.NET Core Components
 
-**Note:** You also need to run the preceding `build` command in the command line before building in VS to ensure that the Web.JS dependency is built.
+1. You'll need to install [Node](https://nodejs.org) on your machine.
 
-### A Guide to build Web.JS
+1. Ensure the repository is clean from any asset that could remain from previous version of the repository. This is recommended when switching branches, or after updating the working branch.
 
-Prior to building `src\Components\Web.JS\`
+```powershell
+git clean -xdff
+```
 
-1. You'll need to install [Node](https://nodejs.org) on your machine.
+You may need to kill some processes holding on files that are being deleted. The following command may help you but be aware that this could also stop other important processes:
+
+```powershell
+Get-Process dotnet, escape-node-job, msbuild, VBCSCompiler, node, vstest.console, Microsoft.CodeAnalysis.LanguageServer -ErrorAction Continue | Stop-Process;
```

Comment:
I usually use `taskkill`, but this should be similar (and cross-plat).

Shutting down VS before cleaning your repo is usually a good idea.

Reply:
Thanks, done